### ISSUE TITLE
CV inputs of Poly Seven Seas are not poly

### ISSUE THREAD

**Issue by smerkley** [nysthi/nysthi#191]
Not sure if feature request or bug, probably you already know this, the only CV inputs on Poly Seven Seas that work with poly are the FM and V/Oct. I'd love to be able to morph the wavetable with poly.

**Comment by nysthi** [comment]
you'll need a quantistic computer...

**Comment by novembernine** [comment]
I'm happily able to play 2 Poly Seven Seas modules with 16 voice poly, 8 swarm voices and Linear FM in both directions between each module - is it really not viable to have polyphonic modulation for Waves and Banks?  

I appreciate that there may be visualisation problems to solve..

**Comment by nysthi** [comment]
WOW!

I think can be done
but needs an huge effort by me (that I have a real life consuming job)

at least 2 months only to implement this new features

and sincerely I don't feel I can do it

**Comment by novembernine** [comment]
Yes, that would be an unrealistic proposition, sadly!  Already you have given a _heroic_ effort towards pushing forward the Rack platform, I would just like to applaud you for that.

My poly patch now has 2 Poly Seven Seas being modulated by 2 Bogaudio FM-OPs each (with Vult providing various filtering options) - and these are the best sounds I have ever heard coming from a computer!  Mono-modulation of Wave and Bank still provides lots of useful places to go :-)

**Comment by nysthi** [comment]
I generally use unity mixer when I'm going to modulate in poly7seas
or a summer + using the vca

**Comment by novembernine** [comment]
Yes - this is my technique also.  As long as the modulation is relatively slow, it provides a sense of movement to all voices without being so obviously mono-driven :-)

**Comment by nysthi** [comment]
yes, it's kind of paraphonic modulation
